.github/workflows/build-docs.yml

```diff
@@ -1,17 +1,78 @@
 ---
 # Workflow to build the documentation.
 #
-#  On pull requests, partial builds are run except
-#  if "[full doc]" is defined in commit message.
-#  Full builds are always run on "main".
-#  This is done every time there is a push on "main" and every week.
+# - validate CITATION.CFF file
+# - check for dead links
```

Comment:
Sorry, what is the dev supposed to do on these individual items ?

Reply:
> these individual items

what do you mean by those "individual items"?

the top comment of the yml file should be treated like a "docstring"  in rst syntax that will be displayed in the doc

Reply:
Sorry I misunderstood that part. Clear now after seeing the build